CONTRIBUTING.md

```diff
@@ -89,7 +89,9 @@ for more detail on available tox commands.
 
 ### Troubleshooting
 
-> Some packages may require additional system wide dependencies to be installed. For example, you may need to install `libpq-dev` to run the postgresql client libraries instrumentation tests. or `libsnappy-dev` to run the prometheus exporter tests. If you encounter a build error, please check the installation instructions for the package you are trying to run tests for.
+Some packages may require additional system-wide dependencies to be installed. For example, you may need to install `libpq-dev` to run the postgresql client libraries instrumentation tests or `libsnappy-dev` to run the prometheus exporter tests. If you encounter a build error, please check the installation instructions for the package you are trying to run tests for. 
+
+For `docs` building, you may need to install `mysql-client` and other required dependencies as necessary. Ensure the Python version used in your local setup matches the version used in the CI to maintain compatibility when building the documentation.
```

Comment:
I bumped into this the other day and figured it out by searching Slack and finding a thread that points to [this line](https://github.com/open-telemetry/opentelemetry-python/blob/main/.github/workflows/misc_0.yml#L121). Could this paragraph include a hyperlink to maybe the [workflows dir](https://github.com/open-telemetry/opentelemetry-python/tree/main/.github/workflows)?

Reply:
Sure. Will add